docs/cloud/automations.md

```diff
@@ -49,6 +49,7 @@ On the **Automations** page, select the **+** icon to create a new automation. Y
 Triggers specify the conditions under which your action should be performed. Triggers can be of several types, including triggers based on: 
 
 - Flow run state change
+    -  Note - Flow Run Tags currently are only evaluated with `OR` criteria
```

Comment:
@WillRaphaelson Do you think there's a place in here or elsewhere for something like "looking to run code in response to flow run state changes? Look to our (brand new) documentation on state change hooks [link here]."

Reply:
Sure - and perhaps a small discussion of when one might use which. state change hooks exist in OSS for example, and are only on the state change, not the _lack of_ state change which is where automations really shine.